src/_preflights/resets.js

```diff
@@ -0,0 +1,4 @@
+export const resets = {
+  layer: 'preflights',
+  getCSS: async () => (await fetch('https://assets.finn.no/pkg/@warp-ds/css/v1/resets.css')).text(),
```

Comment:
I just noticed that we also make a minified version available on eik ([@warp-ds/css PR](https://github.com/warp-ds/css/pull/2/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R11-R17)). There it was mentioned that based on whether it's used in production or in development, the minified or non-minified version can be used. Do you think we can keep the non-minified version here or should we rather replace it with the minified since we're not really saying anywhere that we're in development mode? Just checking, I don't mind either option.

Reply:
Exactly, I've added to have discussion about it. Don't think I added any comment about the discussion though 🤦 Sorry. I **_think_** that we should have minified version here. And then fix so we can use non minified version in dev. But then maybe we will need a dev flag for it? 🤔 
Maybe the non minified step is just too much. What do you think?

Reply:
If we do keep dev flag, we'd need to find some smart way of using that to conditionally return minified or non-minified version of the resets.css. The only solution I could think of from the top of my head would be to export two constants from this file, `minResets` and `resets`, and then apply one of them based on `options.development`. But I'm sure there are smarter ways 😄

Reply:
Shall we do it you think? I'm starting leaning towards only having the minified version 😅 And if we will need the unminified one then we can add it. Preflights are very static and won't change a lot I suppose so maybe there is no point to complicate the setup for that 🤔 wdyt?

Reply:
I totally agree! Let's keep the minified version only until suggested otherwise 👍

Reply:
Changed here https://github.com/warp-ds/drive/pull/106/commits/274a6f7cd8f21f21d77baa83171d63698669b83e. Awaiting css repo changes so the minified version can be published